Library/Homebrew/.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
Can you remove this? Thanks! Might be worth adding `.idea` so your system-wide `~/.gitignore`